internals/cli/cli.go

```diff
@@ -52,6 +52,14 @@ const defaultPebbleDir = "/var/lib/pebble/default"
 // ErrExtraArgs is returned  if extra arguments to a command are found
 var ErrExtraArgs = fmt.Errorf("too many arguments for command")
 
+// CmdOptions exposes state made accessible during command execution.
+type CmdOptions struct {
+	// Client is the Pebble client instance
+	Client *client.Client
+	// Parser is the instance of the flags.Parser that processed the command.
```

Comment:
Comments should add information which is not obvious, otherwise they're a negative asset: adds noise and work for readers, with no extra value. The two examples above are typical cases of this, as we're literally saying "foo is an instance of Foo". Can we please just drop both?